src/OrchardCore/OrchardCore.DisplayManagement.Liquid/ViewBufferTextWriterContent.cs

```diff
@@ -315,6 +343,27 @@ public override Task WriteLineAsync(StringBuilder value, CancellationToken cance
             return Task.CompletedTask;
         }
 
-        #endregion
+        public override string ToString()
+        {
+            using var pool = StringBuilderPool.GetInstance();
+            using var sw = new StringWriter(pool.Builder);
+            WriteTo(sw, NullHtmlEncoder.Default);
+            return sw.ToString();
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private void CheckDisposed()
+        {
+            if (_disposed)
```

Comment:
More than `_disposed` it should actually check for `_released` or just throw an exception in the `WriteTo` method if it's called twice, or already released.